install_config/topics/red_hat_openshift_container_storage.adoc

```diff
@@ -0,0 +1,11 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_aws.adoc
+install_config/configuring_azure.adoc
+install_config/configuring_gce.adoc
+install_config/configuring_vmware.adoc
+////
+
+= Configuring OpenShift Container Platform for OpenShift Container Storage 
+Red Hat OpenShift Container Storage (RHOCS) is a provider of agnostic persistent storage for OpenShift Container Platform (OCP) on-premise and in hybrid clouds, As a Red Hat storage solution, RHOCS is completely integrated with OCP for deployment, management and monitoring regardless if it is installed on top of OpenShift Container Platform (converged) or alongside OpenShift Container Platform (independent). OpenShihft Container Storage is not tied down to a single availability zone or node making it very attractive for surviving an outage. Complete instructions for using RHOCS can be found in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/3.11/html-single/deployment_guide/RHOCS[3.11 Deployment Guide].
```

Comment:
I'd replace lines 10 and later with something like this:

[id='about-RHOCS-{context}']
=  About Red Hat OpenShift Container Storage 
Red Hat OpenShift Container Storage (RHOCS) is a provider of agnostic persistent storage for {product-title} either in-house or in hybrid clouds. As a Red Hat storage solution, RHOCS is completely integrated with {product-title} for deployment, management, and monitoring regardless if it is installed on {product-title} (converged) or with {product-title} (independent). OpenShift Container Storage is not limited to a single availability zone or node, which makes it likely to survive an outage. You can find complete instructions for using RHOCS in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/3.11/html-single/deployment_guide/RHOCS[3.11 Deployment Guide].

That link doesn't work for me.